client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -154,33 +155,30 @@ export const CreateEncounterModal: FC = () => {
                   label={t('label.visit-date')}
                   Input={
                     <DatePickerInput
-                      value={DateUtils.getDateOrNull(
-                        draft?.startDatetime ?? null
-                      )}
+                      value={draft?.startDatetime}
```

Comment:
Moved the conversion from string to Date object into the actual Picker components -- this prevents another one of those `useEffect` errors where it thinks a value has changed but it actually hasn't, it just got a new instance of the same thing.